messaging/broker.go

```diff
@@ -144,943 +222,944 @@ func (b *Broker) closeTopics() {
 	for _, t := range b.topics {
 		_ = t.Close()
 	}
-	b.topics = make(map[uint64]*topic)
+	b.topics = make(map[uint64]*Topic)
 }
 
-// closeReplicas closes all replica writers and clears the replica map.
-func (b *Broker) closeReplicas() {
-	for _, r := range b.replicas {
-		r.closeWriter()
-	}
-	b.replicas = make(map[uint64]*Replica)
+// SetMaxIndex sets the highest index seen by the broker.
```

Comment:
"seen" is not a Raft term. Is there a specific Raft term that you actually mean here? "Committed", "applied"?